### PR TITLE
Compress Json Files - Public Repo

### PR DESCRIPTION
**Pull request by Shellyber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/33657

## Description
Compressed the following files:
* id_set.json
* corepacks.json
* index.json

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Comment by Shellyber** [comment]
https://console.cloud.google.com/storage/browser/marketplace-ci-build/content/builds/compress-files/211d65e9-1c42-4d3d-a1f0-5f29a70cb353/content;tab=objects?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=core&forceOnObjectsSortingFiltering=false

Example for the latest run.
See the index.json, corepacks.json and id_set.json.
The file size is lowered significantly.

**Review by guyfreund — CHANGES_REQUESTED** [review]
Very Nice Work! 👏

File: Tests/Marketplace/upload_packs.py [hunk 1]
```diff
@@ -1105,9 +1111,6 @@ def main():
                             storage_bucket=storage_bucket,
                             )
 
-    # upload id_set.json to bucket
-    upload_id_set(storage_bucket, id_set_path)
-
```
[the remark this answers is not in the document]
Comment (Shellyber) [reply Tests/Marketplace/upload_packs.py:1110]: I'll to it after handling the private. If I'll do it know, private will fail

File: Tests/Marketplace/upload_packs.py [hunk 2]
```diff
@@ -369,17 +368,23 @@ def upload_core_packs_config(storage_bucket: Any, build_number: str, index_folde
...
-    blob = storage_bucket.blob(core_packs_config_path)
-    blob.upload_from_string(json.dumps(core_packs_data, indent=4))
+    corepacks_json_path = os.path.join(index_folder_path, f'{GCPConfig.CORE_PACK_FILE_NAME}')
+    with open(corepacks_json_path, 'w+') as corepacks_file:
+        # construct core pack data with public gcs urls
+        core_packs_data = {
+            'corePacks': core_packs_public_urls,
+            'buildNumber': build_number
+        }
+        json.dump(core_packs_data, corepacks_file, indent=4)
```
[the remark this answers is not in the document]
Comment (Shellyber) [reply Tests/Marketplace/upload_packs.py:379]: Reverted this change since the `json_write` uses json.dumps and not dump

**Comment by Shellyber** [comment]
@guyfreund Please review.
Last upload test run: https://console.cloud.google.com/storage/browser/marketplace-ci-build/content/builds/compress-files/43e3a76a-79f2-4115-8d02-1b15a572d36a/content?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false

**Review by guyfreund — COMMENTED** [review]
please attach the build run

**Comment by Shellyber** [comment]
@guyfreund Latest build run: https://app.circleci.com/pipelines/github/demisto/content/90447/workflows/43e3a76a-79f2-4115-8d02-1b15a572d36a

**Comment by Shellyber** [comment]
This is the private commit bucket - https://console.cloud.google.com/storage/browser/marketplace-ci-build-private/content/builds/compress-private-files/3994/content?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false

See the size of the id_set.

**Comment by Shellyber** [comment]
@guykeller FYI
Removed the `upload_id_set` command. We are now using `gsutil` to compress the size.

**Review by guyfreund — APPROVED** [review]
🥇

**Pull request merged**